### PR TITLE
Session Persistence update fix

### PR DESCRIPTION
**Pull request by mdurrant-b3**
Session Persistence update fix
* Added logic to test for when persistence changes
* Correctly handles create/update/delete

### CODE REVIEW

**Comment by mdurrant-b3** [comment]
check

**Comment by dougwig** [comment]
check

**Comment by mdurrant-b3** [comment]
check

**Comment by dougwig** [comment]
check

File: a10_neutron_lbaas/tests/unit/v2/test_handler_listener.py [hunk 1]
```diff
@@ -81,12 +81,15 @@ def test_create(self):
                         self.assertTrue(test_prot in s)
 
                     if pers == 'SOURCE_IP':
+                        self.assertTrue('c_pers=SOURCE_IP')
```
Comment (dougwig) [inline a10_neutron_lbaas/tests/unit/v2/test_handler_listener.py]: These three asserts do nothing.

File: a10_neutron_lbaas/tests/unit/v1/test_handler_vip.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2014, Doug Wiegley (dougwig), A10 Networks
+# Copyright 2016 A10 Networks
```
Comment (dougwig) [inline a10_neutron_lbaas/tests/unit/v1/test_handler_vip.py]: 2014-2016

File: a10_neutron_lbaas/v2/handler_pool.py [hunk 3]
```diff
@@ -84,3 +83,35 @@ def delete(self, context, pool):
...
+            p.create()
+            return
+
+        # didn't exist, does exist now, create
+        if old_pool.session_persistence and not pool.session_persistence:
+            p = handler_persist.PersistHandler(c, context, pool)
+            p.create()
+            return
+
+        # didn't exist, doesn't exist
```
Comment (dougwig) [inline a10_neutron_lbaas/v2/handler_pool.py]: from 110 down, you can just put in the two comments and not the if conditionals, as they do nothing.

**Pull request merged**